base/logging.go

```diff
@@ -248,6 +248,14 @@ func LogNoTime() {
 	logNoTime = true
 }
 
+func LogTime() {
+	logLock.RLock()
+	defer logLock.RUnlock()
+	//Disable timestamp for default logger, this may be used by other packages
```

Comment:
Is this comment correct, or is it copy-pasted from above and should read 'Enable'?

Reply:
Changed to Enable